css/index.css

```diff
@@ -102,11 +125,116 @@ p {
 }
 
 .middle-content .boxes .box {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
On these boxes, the parent can take care of everything except the color (which you need to override for the child).  It saves many lines of code and is easier to read.  Remember the only difference between these individual boxes is the color.